Argo/Globals/JSONValue.swift

```diff
@@ -1,24 +1,24 @@
 import Foundation
 
+public typealias JSONDict = [String: JSONValue]
+
 public enum JSONValue {
-  case JSONObject([String:JSONValue])
+  case JSONObject(JSONDict)
   case JSONArray([JSONValue])
   case JSONString(String)
   case JSONNumber(NSNumber)
   case JSONNull
+}
 
-  public static func parse(json: AnyObject) -> JSONValue {
+public extension JSONValue {
+  static func parse(json: AnyObject) -> JSONValue {
     switch json {
     case let v as [AnyObject]: return .JSONArray(v.map(parse))
 
-    case let v as [String:AnyObject]:
-      var object: [String:JSONValue] = [:]
-      for key in v.keys {
-        if let value: AnyObject = v[key] {
-          object[key] = parse(value)
-        } else {
-          object[key] = .JSONNull
-        }
+    case let v as [String: AnyObject]:
```

Comment:
`JSONDict` is `[String: JSONValue]`. This is `[String: AnyObject]`. I think it's actually just an `NSDictionary` that's being bridged to `Dictionary`.